### PR TITLE
Error messages

### PR DESCRIPTION
**Pull request by vrjuliao**

### CODE REVIEW

File: src/test/kotlin/unit/ParsingTest.kt [hunk 1]
```diff
@@ -73,4 +73,44 @@ class ParsingTest {
     """
     assertNotNull(parse(tokenize(program)))
   }
+
+  @Test
+  @DisplayName("Invalid import statement - without comma")
+  fun invalidSyntaxWithoutComma(){
```
Comment (Yowgf) [inline src/test/kotlin/unit/ParsingTest.kt]: "without" -> "missing"

File: src/test/kotlin/unit/ParsingTest.kt [hunk 2]
```diff
@@ -73,4 +73,44 @@ class ParsingTest {
...
+        };
+    """
+    val error = "line 4:6 missing ';' at 'data'"
+    val e1 = assertFailsWith<ParseCancellationException>{parse(tokenize(program))}
+    assertEquals(error, e1.message)
+  }
+
+  @Test
+  @DisplayName("Invalid EXCEPT - without parenthesis")
+  fun invalidSyntaxWithoutParenthesis(){
```
Comment (Yowgf) [inline src/test/kotlin/unit/ParsingTest.kt]: "without parenthesis" -> "missing curly braces"

**Review by Yowgf — COMMENTED** [review]
Just renaming unit tests

**Comment by vrjuliao** [comment]
Thank you @Yowgf , It's done.

**Pull request merged**